test/e2e_node/cpu_manager_test.go

```diff
@@ -313,6 +315,10 @@ func runNonGuPodTest(f *framework.Framework, cpuCap int64) {
 
 	ginkgo.By("checking if the expected cpuset was assigned")
 	expAllowedCPUsListRegex = fmt.Sprintf("^0-%d\n$", cpuCap-1)
```

Comment:
is the assumption that cpuCap is a single digit number?

Reply:
yep, it passed to the method as `cpuCap int64`

Reply:
as long as this assumption is OK